store/sql_post_store.go

```diff
@@ -386,6 +388,12 @@ func (s SqlPostStore) Search(teamId string, userId string, terms string, isHasht
 		var posts []*model.Post
 
 		if utils.Cfg.SqlSettings.DriverName == "postgres" {
+
+			// Parse text for wildcards
+			if wildcard, err := regexp.Compile("\\*($| )"); err == nil {
+				terms = wildcard.ReplaceAllLiteralString(terms, ":* ")
```

Comment:
does this work when it's the end of the line?  my search term is "corey*" vs "corey* "